src/main/jbake/content/blog/2017/01/13/events-aggregation-extension.adoc

```diff
@@ -0,0 +1,167 @@
+= Extending Complex Event Processing in Hawkular Alerting
+Lucas Ponce
+2017-01-13
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, alerts, complex, events, processing, stream, polling
+
+== Stream Processing versus Polling Processing
+
+Hawkular Alerting uses different techniques to detect behaviours by defining rules.
+
+The main implementation is based in a Stream Processing design. +
```

Comment:
Currently these lines are all on new lines, although they appear to be part of a paragraph. Would suggest either adding a line separator, or removing the line breaks so they are formatted as a paragraph.